apps/web/src/components/Calendar/body/month/calendar-body-month.tsx

```diff
@@ -1,22 +1,24 @@
 import React from 'react'
 import { useTranslation } from 'react-i18next'
 import { useCalendarContext } from '../../calendar-context'
-import { DateTime, Interval, Info } from 'luxon'
+import { Interval, Info } from 'luxon'
+import { DateTimeHelpers } from '@hylo/shared'
 import { cn } from '@/lib/utils'
 import CalendarEvent from '../../calendar-event'
 import { AnimatePresence, motion } from 'framer-motion'
-import { eachIntervalDay, getLocaleAsString, includes, sameDay, sameMonth } from '../../calendar-util'
+import { eachIntervalDay } from '../../calendar-util'
+import { localeLocalStorageSync } from 'util/locale'
 
 export default function CalendarBodyMonth () {
   const { t } = useTranslation()
   const { date, events, setDate, setMode } = useCalendarContext()
-  const luxonDate = DateTime.fromJSDate(date)
+  const luxonDate = DateTimeHelpers.toDateTime(date, { locale: localeLocalStorageSync() })
```

Comment:
localeLocalStorageSync is a bit of an odd name, why not getLocalFromLocalStorage?